src/bootstrap/lib.rs

```diff
@@ -532,8 +532,13 @@ impl Build {
     fn std_features(&self) -> String {
         let mut features = "panic-unwind".to_string();
 
-        if self.config.llvm_libunwind {
-            features.push_str(" llvm-libunwind");
+        match self.config.llvm_libunwind.as_deref() {
+            Some("in-tree") => features.push_str(" llvm-libunwind"),
+            Some("system") => features.push_str(" system-llvm-libunwind"),
+            Some("no") | None => {}
+            Some(invalid) => {
+                panic!("Invalid value '{}' for rust.llvm-libunwind config.", invalid);
```

Comment:
Please move this parsing/validation to config.rs; probably we want to add an enum with three variants (No, System, InTree).